src/Protocol/OStatus.php

```diff
@@ -1930,6 +1930,10 @@ private static function entryFooter(DOMDocument $doc, $entry, array $item, array
 
 		if ($item['gravity'] != GRAVITY_PARENT) {
 			$parent = Post::selectFirst(['guid', 'author-link', 'owner-link'], ['id' => $item['parent']]);
+			if (!DBA::isResult($parent)) {
```

Comment:
The `parent` field is no "real" field but is is build in the view by the `id` of the parent post. This means that it has to exist. So when this query here fails then there has to be some underlying problem, possibly with the database configuration. I wouldn't want to handle it this way.